hbase/src/main/java/com/yahoo/ycsb/db/HBaseClient10.java

```diff
@@ -92,6 +94,9 @@
      */
     public boolean _clientSideBuffering = false;
     public long _writeBufferSize = 1024 * 1024 * 12;
+    public int multiGetsCount = 0;
```

Comment:
@busbey - I think that is the number of gets to put in each request. maybe a better name for it is multiGetBatchSize or maxGetBatchSize?